x-pack/plugins/ml/public/application/services/ml_api_service/trained_models.ts

```diff
@@ -57,6 +58,17 @@ export interface InferenceStatsResponse {
  */
 export function trainedModelsApiProvider(httpService: HttpService) {
   return {
+    /**
+     * Fetches the trained models list available for download
+     */
+    getTrainedModelDownloads() {
+      return httpService.http<ModelDefinitionResponse[]>({
+        path: `${ML_INTERNAL_BASE_PATH}/trained_models/model_downloads`,
+        method: 'GET',
+        version: '1',
```

Comment:
Should we make this 2 immediately? This change will only go into the 8.11 branch, and we'd like to see how things work with v2 ASAP in that branch. Or am I missing the significance of this being 1?

Reply:
This is the api version. As this is the first version of this `/trained_models/model_downloads` endpoint, the version should be `1`